libraries/ms-common/inc/uart.h

```diff
@@ -3,14 +3,22 @@
 // Requires GPIO and interrupts to be initialized.
 
 // Uses internal FIFOs to buffer RX and TX.
+#include <fcntl.h>
```

Comment:
Don't add unnecessary includes to the header. The header is a generic interface.

Reply:
Done

Reply:
<fcntl.h> includes the open-time flags O_NOCTTY and O_RWDR

Reply:
I could define the flags locally but would that not be bad practice?

Reply:
No, just include them in your source file. You shouldn't be exposing the need for any of these includes in application code.

Reply:
Can you please remove any changes you've made to the header?